File: python-version/LC1656/ordered_stream_test.py
import unittest
from ordered_stream import OrderedStream

class OrderedStreamTest(unittest.TestCase):
    def test_orderedStream(self):
        stream = OrderedStream(5)
        res = []
        res.append(stream.insert(3, "ccccc"))
        res.append(stream.insert(1, "aaaaa"))
        res.append(stream.insert(2, "bbbbb"))
        res.append(stream.insert(5, "eeeee"))
        res.append(stream.insert(4, "ddddd"))

if __name__ == '__main__':
    unittest.main()

File: java-version/src/test/java/com/github/leetcode/LC1656/OrderedStreamTest.java
package com.github.leetcode.LC1656;

import com.github.leetcode.LC59.Solution;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class OrderedStreamTest {

    @Test
    void insert() {
        OrderedStream stream = new OrderedStream(5);
        System.out.println(stream.insert(3, "ccccc"));
        System.out.println(stream.insert(1, "aaaaa"));
        System.out.println(stream.insert(2, "bbbbb"));
        System.out.println(stream.insert(5, "eeeee"));
        System.out.println(stream.insert(4, "ddddd"));
    }
}
